v2/pages/adminAssessment/adminAssessment.sql

```diff
@@ -23,6 +23,44 @@ WHERE
     AND q.deleted_at IS NULL
 ORDER BY z.number, z.id, aq.number;
 
+-- BLOCK question_stats
+SELECT
+    sub.question_id,
+    avg(sub.mean_score_per_user_per_question) as mean_score_per_question,
+    sub.number,
+    corr(sub.mean_score_per_user_per_question, sub2.average_assessment_score_for_user) as discrimination
+FROM (
+    SELECT
+        assessment_instances.user_id,
+        assessment_questions.question_id,
+        assessment_questions.number,
+        avg(instance_questions.points/assessment_questions.max_points) as mean_score_per_user_per_question
+    FROM
```

Comment:
I still think that this would be easier to read if you put `sub` and `sub2` as `WITH` queries before the main `SELECT` and called them something like `mean_question_scores` and `mean_assessment_scores` with columns like `score_perc`. But this isn't a big deal either way.